appveyor.yml

```diff
@@ -20,8 +21,11 @@ test_script:
   # Output useful info for debugging.
   - node --version
   - node_modules\.bin\npm --version
-  # run tests
-  - node_modules\.bin\npm run builder:check
+  # TODO REENABLE
```

Comment:
Haha, it's there precisely to make sure I **don't** leave this in the final PR (I had a lot of windows debugging to do earlier...)